### PR TITLE
2023 rewrite

### PR DESCRIPTION
**Pull request by ErinBecker**
Update information to be relevant for 2023 Maintainer cohort. Details of changes for each episode are listed in the individual commits.

### CODE REVIEW

**Comment by ErinBecker** [comment]
@tobyhodges @vinisalazar - If you have a chance before the first Maintainer onboarding session, please take a look at the revisions here. My current plan is to teach episodes 1-5, and skip the "Git Workflow" episode.

File: episodes/05-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+:::::::::::::::::::::::::::::::::::::::
+
+```
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+This is a challenge.
+
+:::::::::::::::::::::::::::::::::::::::
```
Comment (tobyhodges) [inline episodes/05-infrastructure.md:145]:
```suggestion
Callouts with the tag "challenge" will render as exercises:

````
::::::::::::::::::::::::::::::::::::::: challenge

This is a challenge.

:::::::::::::::::::::::::::::::::::::::

````

::::::::::::::::::::::::::::::::::::::: challenge

This is a challenge.

:::::::::::::::::::::::::::::::::::::::

Another block with the tag "solution" can be nested inside a challenge to provide the solution to an exercise:

````
::::::::::::::::::::::::::::::::::::::: challenge

What is the solution to this exercise?

::::::::::::::::::::::::::::::::::::::::::::: solution

This is the solution to this exercise.

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::

````

::::::::::::::::::::::::::::::::::::::: challenge

What is the solution to this exercise?

::::::::::::::::::::::::::::::::::::::::::::: solution

This is the solution to this exercise.

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::

```

Comment (tobyhodges) [reply episodes/05-infrastructure.md:145]: 😆 ok looks like GitHub's suggestion feature does not play nicely with Markdown code blocks.

Comment (tobyhodges) [reply episodes/05-infrastructure.md:145]: I had to make this suggestion as a PR to your branch instead: https://github.com/carpentries/maintainer-onboarding/pull/74

File: episodes/05-infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+```
+---
+title: "What is the shell?"
+teaching: 5
+exercises: 0
+---
+```
+
+The information stored in the YAML header is used by the lesson infrastructure to populate the schedule.
```
Comment (tobyhodges) [inline episodes/05-infrastructure.md:101]:
```suggestion
`title` sets the title of the episode, which will be displayed as a top-level heading on the webpage. `teaching` and `exercises` are values (in minutes) indicating how much time the episode is estimated to take to teach, and for learners to complete its exercises. The information stored in the YAML header is used by the lesson infrastructure to populate the schedule.
```

File: episodes/05-infrastructure.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+The Instructor view of the lesson homepage also includes a schedule showing the individual episodes included in the lesson,
+and the time allotted for each. The schedule is automatically created based on information present in
+the episode files.
+
+### Episodes
+
+The majority of a lessons content is in its episode files. Episode files are stored in the
+`_episodes/` folder within your lesson repo (or in `_episodes_rmd/` for lessons written in R).
+Episode files are written in *Markdown* or *RMarkdown*. The essential elements of an episode
+are mentioned below. For complete information about how to contrust each element
```
Comment (tobyhodges) [inline episodes/05-infrastructure.md:86]:
```suggestion
are mentioned below. For complete information about how to contruct each element
```

**Comment by tobyhodges** [comment]
Thank you @ErinBecker, I made some changes to the Infrastructure episode, and I am merging now.

**Pull request merged**